langchain/package.json

```diff
@@ -1204,6 +1200,7 @@
   "dependencies": {
     "@anthropic-ai/sdk": "^0.6.2",
     "ansi-styles": "^5.0.0",
+    "axios": "1.5.1",
```

Comment:
We probably shouldn't propose changing the axios version, this would impact all of langchain

Reply:
I will update that soon.

Reply:
I changed that version to previous one and configure json api with that